### PR TITLE
Update to krona tools

### PR DESCRIPTION
**Pull request by nekrut**
Updated krona tool as suggested by @nsoranzo

### CODE REVIEW

File: tools/taxonomy_krona_chart/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<tool_dependency>
+  <package name="package_krona_2_5" version="2.5">
```
Comment (bgruening) [inline tools/taxonomy_krona_chart/tool_dependencies.xml]: This should be `krona` according to https://github.com/galaxyproject/tools-iuc/blob/master/packages/package_krona_2_5/tool_dependencies.xml

File: tools/taxonomy_krona_chart/taxonomy_krona_chart.xml [hunk 2]
```diff
@@ -1,32 +1,113 @@
-<tool id="taxonomy_krona_chart" name="Krona pie chart" version="1.0.0">
+<tool id="taxonomy_krona_chart" name="Krona pie chart" version="2.0.0">
   <description>from taxonomic profile</description>
   <requirements>
-    <requirement type="package" version="2.4">kronatools</requirement>
+    <requirement type="package" version="2.5">package_krona_2_5</requirement>
```
Comment (bgruening) [inline tools/taxonomy_krona_chart/taxonomy_krona_chart.xml]: Just `krona` should be enough.

File: tools/taxonomy_krona_chart/tool_dependencies.xml (hunk 1 above)
Comment (martenson) [inline tools/taxonomy_krona_chart/tool_dependencies.xml]: sorry @nekrut  :)

File: tools/taxonomy_krona_chart/taxonomy_krona_chart.xml [hunk 3]
```diff
@@ -1,32 +1,113 @@
...
+  #end if
+
+  ]]>
+
   </command>
   <inputs>
-    <param name="input" type="data" format="tabular" label="Input file" help="Either a 'MetaPhlAn to Krona' result file or a taxonomy file" />
+    <conditional name="type_of_data">
+      <param name="type_of_data_selector" type="select" label="What is the type of your input data" help="Select between Galaxy Taxonomy and MetaPhlAn Text">
+      <option value="taxonomy" selected="True">Taxonomy</option>
```
Comment (bgruening) [inline tools/taxonomy_krona_chart/taxonomy_krona_chart.xml]: indentation ...

**Comment by nsoranzo** [comment]
Apart from bgruening's concerns, :+1: 

Thanks @nekrut!

**Comment by nekrut** [comment]
@nsoranzo @bgruening @erasche @jmchilton please, accept

**Pull request merged**